src/libs/ValidationUtils.ts

```diff
@@ -354,8 +355,8 @@ function isReservedRoomName(roomName: string): boolean {
 /**
  * Checks if the room name already exists.
  */
-function isExistingRoomName(roomName: string, reports: Record<string, Report>, policyID: string): boolean {
-    return Object.values(reports).some((report) => report && report.policyID === policyID && report.reportName === roomName);
+function isExistingRoomName(roomName: string, reports: OnyxCollection<Report>, policyID?: string): boolean {
```

Comment:
```suggestion
function isExistingRoomName(roomName: string, reports: OnyxCollection<Report>, policyID: string | undefined): boolean {
```

This one makes sense to be `string | undefined` since we can pass an `undefined` but we don't want to omit the argument.